src/components/Login.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useAuth } from '../contexts/AuthContext';
+
+export function Login() {
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
export function Login() {
  const [isLoading, setIsLoading] = useState(false);
  const { signInWithGoogle } = useAuth();

  const handleSignIn = async () => {
    setIsLoading(true);
    try {
      await signInWithGoogle();
    } catch (error) {
      console.error('Authentication failed:', error);
    } finally {
      setIsLoading(false);
    }
  };
```
Add loading state management to prevent multiple sign-in attempts and provide visual feedback during authentication



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;